go/vt/mysqlctl/rice-box.go

```diff
@@ -29,9 +29,9 @@ func init() {
 	}
 	file6 := &embedded.EmbeddedFile{
 		Filename:    "mycnf/default.cnf",
-		FileModTime: time.Unix(1579019403, 0),
+		FileModTime: time.Unix(1579632282, 0),
```

Comment:
Are we planning to remove this file from source control? since it changes every time..

Reply:
This is the intended behavior. In my first PR that proposed this rice box, it was only generated by the makefile. Sugu pointed out that this breaks go get (and is not a go-idiom) :-)

It no longer regenerates on any make build though: so it should not be spammy any more. This is an intended behavior here, because I ran `make embed_config` explicitly after editing the file.